listenbrainz/config.py.sample

```diff
@@ -119,6 +119,7 @@ API_URL = 'https://api.listenbrainz.org'
 LASTFM_PROXY_URL = 'http://localhost:8080/'
 SERVER_ROOT_URL = 'http://localhost'
 WEBSOCKETS_SERVER_URL = 'http://localhost:8081'
+PLAYLIST_WEBSOCKETS_URL = 'http://localhost:8082'
```

Comment:
what's the difference between websockets_server and playlist_websockets? We should be using the same server for both playlists and follow if possible

Reply:
I think it is possible but I tried to keep everything separate for easier testing and the follow server is going to be removed soon so there will be only one server.